dashboard/src/deployment/DeploymentList.js

```diff
@@ -1,48 +1,67 @@
 import React, { Component } from 'react';
 import { apiGet } from '../api/api.js';
-import DeploymentRow from './DeploymentRow.js';
-import { Table } from 'semantic-ui-react';
-//import logo from './logo.svg';
-//import './App.css';
+import { Icon, Table } from 'semantic-ui-react';
+import Loading from '../util/Loading.js';
+
+const HeaderView = () => (
+  <Table.Header>
+    <Table.Row>
+      <Table.HeaderCell>State</Table.HeaderCell>
+      <Table.HeaderCell>Name</Table.HeaderCell>
+      <Table.HeaderCell>Mode</Table.HeaderCell>
+      <Table.HeaderCell>Pods</Table.HeaderCell>
+    </Table.Row>
+  </Table.Header>
+);
+
+const RowView = ({info}) => (
+  <Table.Row>
+    <Table.Cell><Icon name="bell" color="red"/></Table.Cell>
+    <Table.Cell>{info.name}</Table.Cell>
+    <Table.Cell>{info.mode}</Table.Cell>
+    <Table.Cell>{info.ready_pod_count} / {info.pod_count}</Table.Cell>
+  </Table.Row>
+);
+
+const ListView = ({items}) => (
+  <Table striped celled>
+    <HeaderView/>
+    <Table.Body>
+      {
+        (items) ? items.map((item) => <RowView key={item.name} info={item}/>) : <p>No items</p>
```

Comment:
i.e. here I'd do `<RowView key={item.name} name={item.name} mode={item.mode} readyCount={item.ready_pod_count} totalCount={item.pod_count} />` so it's clearer what this takes.